### PR TITLE
Proof SYSTEM_TEST_STANDARD_LEAD_INS (fix #261)

### LINKED ISSUE #261: Proof SYSTEM_TEST_STANDARD_LEAD_INS
Proof read transcribed code of [SYSTEM_TEST_STANDARD_LEAD_INS](//github.com/chrislgarry/Apollo-11/blob/master/Comanche055/SYSTEM_TEST_STANDARD_LEAD_INS.agc) against scans

Lines: 148
Page: 0420—0422

Reduced quality scans can be found [here][1]

[1]://ibiblio.org/apollo/ScansForConversion/Comanche055

### PR DESCRIPTION
**Pull request by Kwpolska**

### CODE REVIEW

File: Comanche055/SYSTEM_TEST_STANDARD_LEAD_INS.agc [hunk 1]
```diff
@@ -48,7 +48,7 @@
 #
 #	TC	BANKCALL
 #	CADR	E/BKCALL
-#	CADR	ROUTINE		# WHERE TO WANT TO GO IN FIXED.
+#	CADR	ROUTINE		# WHERE YOU WANT TO GO IN FIXED.
```
Comment (wopian) [inline Comanche055/SYSTEM_TEST_STANDARD_LEAD_INS.agc]: ✔️

File: Comanche055/SYSTEM_TEST_STANDARD_LEAD_INS.agc [hunk 2]
```diff
@@ -68,7 +68,7 @@ E/BKCALL	DXCH	BUF2		# SAVE A,L AND GET DP RETURN.
 		CA	0 -1		# GET CADR OF ROUTINE.
 		TC	SWCALL		# GO TO ROUTINE, SETTING Q TO SWRETURN
 					# AND RESTORING A + L.
-		TC	+4		# TX Q, V34, OR BADD STALL RETURN.
+		TC	+4		# TX Q, V34, OR BAD STALL RETURN.
```
Comment (wopian) [inline Comanche055/SYSTEM_TEST_STANDARD_LEAD_INS.agc]: ✔️

File: Comanche055/SYSTEM_TEST_STANDARD_LEAD_INS.agc [hunk 3]
```diff
@@ -82,7 +82,7 @@ E/SWITCH	DXCH	EBUF2
 #
 #	RTB
 #		E/CALL
-#	CADR	ROUTINE			# THE INTERPRETIVE SUBROUTINE YOU WANT
+#	CADR	ROUTINE			# THE INTERPRETIVE SUBROUTINE YOU WANT.
```
Comment (wopian) [inline Comanche055/SYSTEM_TEST_STANDARD_LEAD_INS.agc]: ✔️

File: Comanche055/SYSTEM_TEST_STANDARD_LEAD_INS.agc [hunk 4]
```diff
@@ -96,22 +96,22 @@ E/CALL		LXCH	LOC		# ADRES -1 OF CADR.
 			EBUF2		# INDIRECTLY EXECUTE ROUTING.  IT MUST
 		EXIT			# LEAVE VIA RVQ OR EQUIVALENT.
 		LXCH	EBUF2 +1	# PICK UP RETURN.
-		TCF	INTPRET +2	# SET LOC AND RETURN TO CALLER.
+		TCF	INTPRET +2	# SET LOC AND RETURN TO CALLER
```
Comment (wopian) [inline Comanche055/SYSTEM_TEST_STANDARD_LEAD_INS.agc]: ✔️ `.` removal
✨ `RETURN` -> `RETURB`

File: Comanche055/SYSTEM_TEST_STANDARD_LEAD_INS.agc [hunk 5]
```diff
@@ -96,22 +96,22 @@ E/CALL		LXCH	LOC		# ADRES -1 OF CADR.
...
+# THE CALLING SEQUENCE IS:
 #
 #	INHINT
 #	...
 #	CA	WAKEADR		# ADDRESS OF SLEEPING JOB
 #	TC	IBNKCALL
 #	CADR	E/JOBWAK
 #	...			# RETURNS HERE
-#	RELINT			# IF YOU DID AND INHINT
+#	RELINT			# IF YOU DID AN INHINT.
```
Comment (wopian) [inline Comanche055/SYSTEM_TEST_STANDARD_LEAD_INS.agc]: ✔️

File: Comanche055/SYSTEM_TEST_STANDARD_LEAD_INS.agc [hunk 6]
```diff
@@ -96,22 +96,22 @@ E/CALL		LXCH	LOC		# ADRES -1 OF CADR.
...
-		TCF	INTPRET +2	# SET LOC AND RETURN TO CALLER.
+		TCF	INTPRET +2	# SET LOC AND RETURN TO CALLER
 
 # Page 422
 # E/JOBWAK	FOR WAKING UP ERASABLE MEMORY JOBS.
 #
 # THIS ROUTINE MUST BE CALLED IN INTERRUPT OR WITH INTERRUPTS INHIBITED.
 #
-# THE CALLING SEQUENCE IS
+# THE CALLING SEQUENCE IS:
```
Comment (wopian) [inline Comanche055/SYSTEM_TEST_STANDARD_LEAD_INS.agc]: ✔️

File: Comanche055/SYSTEM_TEST_STANDARD_LEAD_INS.agc [hunk 7]
```diff
@@ -96,22 +96,22 @@ E/CALL		LXCH	LOC		# ADRES -1 OF CADR.
...
 #
 #	INHINT
 #	...
```
Comment (wopian) [inline Comanche055/SYSTEM_TEST_STANDARD_LEAD_INS.agc]: ✨ Can you change this to

```
#	INHINT
#	  .
#	  .
```

File: Comanche055/SYSTEM_TEST_STANDARD_LEAD_INS.agc [hunk 8]
```diff
@@ -96,22 +96,22 @@ E/CALL		LXCH	LOC		# ADRES -1 OF CADR.
...
 #	CA	WAKEADR		# ADDRESS OF SLEEPING JOB
 #	TC	IBNKCALL
 #	CADR	E/JOBWAK
 #	...			# RETURNS HERE
```
Comment (wopian) [inline Comanche055/SYSTEM_TEST_STANDARD_LEAD_INS.agc]: ✨ And this to:

```
#	CADR	E/JOBWAK
#	  .			# RETURNS HERE
#	  .
#	  .
```

**Comment by wopian** [comment]
Thanks 😄

**Comment by chrislgarry** [comment]
@wopian @Kwpolska thanks!

**Pull request merged**